temp_name_lib/src/star_physics/intensity.rs

```diff
@@ -0,0 +1,71 @@
+use nalgebra as na;
+
+use crate::utils::MathErrors;
+
+enum GridPointStatus{
+    NotAvailable,
+    NotLoaded,
+    Loaded,
+}
+
+enum IntensityType{
+    Total,
+    ContinuumOnly
+}
+
+pub struct Intensity {
+    temp_eff:Vec<f64>,
+    log_g:Vec<f64>,
+    lambda:Vec<f64>,
+    file_name:String,//[Ricardo]: I suppose you have different intensity files.
+    
+    temp_eff_index:usize,
+    log_g_index:usize,
+    lambda_index:usize,
+
+    lambda_lower:f64,
+    lambda_upper:f64,
+    limb_darkening_coef:na::Vector4<f64>,
+    grid_status:GridPointStatus,
+    kind:IntensityType,
```

Comment:
Why is it useful to have the intensity type as part of the struct? Does the code very use this information dynamically? I would have expected somewhere in the code
`continuum_intensities = `
and
`total_intensities = `
so that it's immediately clear what kind of grid it concerns.